pkg/apis/apis/v1alpha1/types_apibinding.go

```diff
@@ -69,6 +69,7 @@ type APIBindingSpec struct {
 	//
 	// +required
 	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="APIExport reference must not be changed"
```

Comment:
Do you know if I can I assume that property changes traverse up? It's what I observe in the e2e test but I am not 100% confident.

Reply:
i.e. this sets the validation rule on the top-level `reference` property only, not on `reference.path` and also not on `reference.exportName`. My question is: is it enough to have the validation rule on the top-level `reference.path` only?

The e2e tests indicate, yes, but I wasn't sure 100% :D

Reply:
If e2e is working, 👍